### PR TITLE
Added sample config for linuxserver/grav

### PR DESCRIPTION
**Pull request by ofersadan85**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: grav.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Version 2022/03/09
+# In order to use this location block you need to edit the default file one folder up and comment out the / location
```
Comment (Roxedus) [inline grav.subfolder.conf.sample:2]: Pretty sure the php location also interferes.

**Review by Roxedus — CHANGES_REQUESTED** [review]
See note

File: grav.subfolder.conf.sample (hunk 1 above)
Comment (ofersadan85) [reply grav.subfolder.conf.sample:2]: When I tested this (been a while) I remember not touching the php location and it worked fine. Would you be able to test to verify if this is needed @Roxedus ?

**Comment by ofersadan85** [comment]
@Roxedus see my question on your review

**Comment by Roxedus** [comment]
I have no environment handy to test this, i just know it was a issue with the organizr, and possibly the heimdall contaieners, which both are PHP based.

**Pull request closed without merging**